### PR TITLE
lts-19.24

### PR DESCRIPTION
**Pull request by ishiy1993**
- Update resolver to lts-19.24.
- Drop old resolvers in CI.
  - Old stack fails to download build plan from stackage.
  - https://github.com/ishiy1993/haiji/actions/runs/3087777392/jobs/4993593686
- Skip doctest for GHC 9.
  - https://github.com/sol/doctest/issues/327
- CI passed
  - https://github.com/ishiy1993/haiji/actions/runs/3095327049

### CODE REVIEW

**Comment by ishiy1993** [comment]
@notogawa Please review :pray:  (I can't request a review from Web UI in this PR.)

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -7,40 +7,39 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    continue-on-error: true
```
Comment (notogawa) [inline .github/workflows/master.yml:10]: required?

Comment (ishiy1993) [reply .github/workflows/master.yml:10]: No.  If it is `false`, when some LTS versions fail, other versions will be cancelled. 
https://github.com/ishiy1993/haiji/actions/runs/3087777392

**Review by notogawa — APPROVED** [review]
LGTM

**Pull request merged**